shared/profile/pgp/pgp-icon.desktop.js

```diff
@@ -0,0 +1,69 @@
+/* @flow */
+
+import React from 'react'
+import {omit} from 'lodash'
+import {Box, Icon} from '../../common-adapters'
+import {globalColors} from '../../styles/style-guide'
+import type {IconType} from '../../common-adapters/icon'
+import type {Props, Overlays} from './pgp-icon'
+
+type OverlayProps = {
+  style: {
+    fontSize: number,
+    color: string,
+    bottom: number,
+    right: number,
+  },
+  type: IconType,
+}
+
+const supportedOverlays: {[key: Overlays]: OverlayProps} = {
+  'good': {
+    style: {
+      fontSize: 24,
+      color: globalColors.green,
+      bottom: -2,
+      right: 5,
+    },
+    type: 'iconfont-proof-good',
+  },
+  'placeholder': {
+    style: {
+      fontSize: 24,
+      color: globalColors.grey,
+      bottom: -2,
+      right: 5,
+    },
+    type: 'iconfont-proof-placeholder',
+  },
+  'import': {
+    style: {
+      fontSize: 16,
+      color: globalColors.green,
+      bottom: -1,
+      right: 11,
+    },
+    type: 'iconfont-import',
+  },
+  'generate': {
+    style: {
+      fontSize: 16,
+      color: globalColors.blue,
+      bottom: -1,
+      right: 11,
+    },
+    type: 'iconfont-add',
+  },
+}
+
+const PgpIcon = ({type}: Props) => {
```

Comment:
@chrisnojima Design is all OOO for the time being. Let's go with this for now and then we can change it out later when they get back if need be.